index.html

```diff
@@ -17,18 +17,20 @@
   <link rel="stylesheet" href="./styles/mobile..css">
 </head>
 <body>
-  <nav class="mobile-nav">
+  <header class="mobile-header">
     <div class="logo">
       <h2 class="name">Rassel</h2>
     </div>
-    <div class="hum">
-      <img src="./images/hum.png" alt="">
-    </div>
-  </nav>
+    <nav class="mobile-nav">
+      <div class="hum">
+        <img src="./images/hum.png" alt="">
+      </div>
+    </nav>
+  </header>
 
   <section class="container">
     <div class="primary-text">
-      <h2 class="greetings">Hey there. I'm Rassel</h2>
+      <h1 class="greetings">Hey there. I'm Rassel</h1>
     </div>
     <div class="support-text">
       <h3 class="activity">I am a Full Stack Developer</h2>
```

Comment:
- [ ] Kindly give a proper space between the two sections and match it with the Figma design. Here is your design looks now:

![readme](https://user-images.githubusercontent.com/70483835/219543561-34d95923-5d4f-4928-aaa0-1c9e78dd6640.png)